src/puppetlabs/puppetdb/mq_listener.clj

```diff
@@ -259,8 +260,9 @@
   [{:keys [command annotations] :as msg} e discard]
   (let [attempt (count (:attempts annotations))
         id      (:id annotations)
+        certname (get-in msg [:payload :certname])
         msg     (annotate-with-attempt msg e)]
-    (log/errorf e "[%s] [%s] Fatal error on attempt %d" id command attempt)
+    (log/error e (i18n/trs "[{0}] [{1}] Fatal error on attempt {2} for ''{3}''" id command attempt certname))
```

Comment:
should we do this in the handle-parse-error and handle-command-discard functions as well?